test/xml.js

```diff
@@ -36,6 +36,33 @@ describe('xml', () => {
 
                 expect(FeedContent).to.equal(xml);
             });
+
+            it('should handle a multiple orders', () => {
```

Comment:
I preserved the grammar error from `it('should handle a multiple fulfillments', () => {`